CHANGELOG.next.asciidoc

```diff
@@ -61,6 +61,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - system/raid metricset now uses /sys/block instead of /proc/mdstat for data. {pull}11613[11613]
 - kubernetes.container.cpu.limit.cores and kubernetes.container.cpu.requests.cores are now floats. {issue}11975[11975]
 - Add statistic option into cloudwatch metricset. If there is no statistic method specified, default is to collect Average, Sum, Maximum, Minimum and SampleCount. {issue}12370[12370] {pull}12840[12840]
+- Change `server_status_path` default setting for nginx module {issue}13806[13806], {pull}14099[14099]
```

Comment:
```suggestion
- Change `server_status_path` default setting for nginx module {issue}13806[13806] {pull}14099[14099]
```

Reply:
I wouldn't consider this as a breaking change. Maybe move this changelog entry under bugs session?

Reply:
Sure 👍

Reply:
Thanks!